downstream/modules/eda/proc-eda-verify-event-streams-work.adoc

```diff
@@ -0,0 +1,33 @@
+[id="eda-verify-event-streams"] 
+
+= Verifying your event streams work
+
+Verify that you can use your event stream to connect to a remote system and receive data.
+
+. Log in to {PlatformNameShort}.
+. From the navigation panel, select {MenuADEventStreams}.
+. Select the event stream that you created to validate connectivity and ensure that the event stream sends data to the rulebook activation. 
+. Verify that the events were received.You can see in the *Events received* field that the event was received. You can also see the header for the event stream that contains details about the event.
++
+image:eda-verify-event-streams.png[Verify event streams work]
++
+If you scroll down in the UI, you can also see the body of the payload with more information about the webhook.  
++
+image:eda-verify-rulebook-attachment.png
++
+The Header and Body sections for the event stream are displayed on the Details page. They will differ based on the vendor who is sending the event. The header and body can be used to check the attributes in the event payload,  which will help you in writing conditions in your rulebook. Specifically, it allows you to look at the unique attributes in the payload, which can be used in conditions. For example:
+//-----
```

Comment:
Is there supposed to be an example here? It looks like it goes to the next step.

Reply:
We were in discussions and I hid that with plans to bring it back, if necessary.